easybuild/tools/utilities.py

```diff
@@ -40,6 +40,7 @@
 
 _log = fancylogger.getLogger('tools.utilities')
 
+INDENT_4SPACES = ' ' * 4
```

Comment:
you add this but don't remove it from anywhere else?

Reply:
Yeah, good point, fixed in cb82587